### PR TITLE
Fixes #10400: Quick search text has gone a hard to read gray

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/10400

### CODE REVIEW

**Comment by RaphaelGauthier** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Can you rebase your pull request please ? :)

**Comment by RaphaelGauthier** [comment]
PR rebased

File: rudder-web/src/main/webapp/style/rudder/rudder-menu.css [hunk 1]
```diff
@@ -307,6 +307,20 @@ div.topQuickSearchResults.ac_results li {
   background-color: rgba(255, 255, 255, 0.65);
   border: none;
 }
+.skin-yellow .sidebar-form.topbar-search-form input.form-control::-webkit-input-placeholder {
```
Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/style/rudder/rudder-menu.css]: You should put it into rudder-quickseach.css

**Comment by RaphaelGauthier** [comment]
Commit modified

**Pull request merged**